### ISSUE TITLE
Blue background used in html publications isn't contrast safe

### ISSUE THREAD

**Issue by edwardhorsford** [alphagov/whitehall#2754]
The blue background used in html publications doesn't meet our minimum contrast requirements.
![screen shot 2016-09-13 at 17 04 16](https://cloud.githubusercontent.com/assets/2204224/18481441/21d695f0-79d4-11e6-9909-6e0335b4b248.png)
Example: https://www.gov.uk/government/publications/budget-2016-documents/budget-2016

This blue is 3.72:1, when we need 4.5:1 to meet WCAG 2.0 AA for small text.

Suggest this blue could be changed to $govuk-blue as an easy fix.

**Comment by boffbowsh** [comment]
Thanks, this is now being tracked in https://trello.com/c/APdc6DYS/699-cardmake-html-publications-contrast-safe
